app/src/NativeBase/Components/SkeletonLoading.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Loading screen showing animated NativeBase Skeleton components to be shown whilst a list is loading.
+ */
+
+import { Skeleton, VStack } from 'native-base'
+import React from 'react'
+
+const SkeletonLoading = () => {
+  return (
+    <>
+      <VStack
+        w="100%"
+        borderWidth="1"
+        space={4}
+        overflow="hidden"
```

Comment:
Could we remove this `overflow` property?  It appears to work the same visually without it (unless there's something I haven't spotted)

Reply:
Done!